src/main/resources/swagger/api-docs.yaml

```diff
@@ -2408,6 +2408,32 @@ paths:
         500:
           description: Internal Server Error
 
+  /api/workspaces/{workspaceNamespace}/{workspaceName}/permissionReport:
+    post:
+      tags:
+        - Method Configurations
+        - Workspaces
+      operationId: workspacePermissionReport
+      summary: Retrieve user permissions for the workspace and the workspace's method references
+      parameters:
+        - $ref: '#/parameters/workspaceNamespaceParam'
+        - $ref: '#/parameters/workspaceNameParam'
+        - name: reportInput
+          description: Users and/or configs on which to report
```

Comment:
Nit - Slightly confusing description. The input can also be an empty json object (`{}`) that will return all users and referenced methods.